app/controllers/users_controller.rb

```diff
@@ -1,3 +1,26 @@
 class UsersController < ApplicationController
-  def new; end
+
+  def show
+    @user = User.find_by id:params[:id]
+  end 
+
+  def new
+    @user = User.new
+  end
+
+  def create
+    @user = User.new user_params
+      if @user.save
+        flash[:success] = "Welcome to the Sample App!"
+        redirect_to @user
+      else
+        render 'new'  
```

Comment:
add 'flash.now' for warning. use double quotes instead of single quote in rb, erb files.